source/_components/orangepi_gpio.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+title: "Orange Pi GPIO"
+description: "Instructions on how to integrate the GPIO capability of a Orange Pi into Home Assistant."
+date: 2019-03-29 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - DIY
+  - Binary Sensor
+ha_release: 0.92
```

Comment:
⚠️ We need to change this to 0.93.
This component is not included in yesterday's cut-off.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/9244